x-pack/metricbeat/module/gcp/metrics/metricset.go

```diff
@@ -107,12 +107,12 @@ type config struct {
 	ExcludeLabels       bool     `config:"exclude_labels"`
 	CredentialsFilePath string   `config:"credentials_file_path"`
 	CredentialsJSON     string   `config:"credentials_json"`
-
-	opt              []option.ClientOption
-	period           *durationpb.Duration
-	organizationID   string
-	organizationName string
-	projectName      string
+	Endpoint            string   `config:"endpoint"`
+	opt                 []option.ClientOption
+	period              *durationpb.Duration
+	organizationID      string
+	organizationName    string
+	projectName         string
```

Comment:
Let's keep a new line between these exposed and internal struct fields like it was before.

```suggestion
	Endpoint            string   `config:"endpoint"`
	
	opt                 []option.ClientOption
	period              *durationpb.Duration
	organizationID      string
	organizationName    string
	projectName         string
```